usr/iscsiadm.c

```diff
@@ -101,6 +101,22 @@ enum _print_node_tree_mode {
 	_PRINT_MODE_NODE,
 };
 
+struct verify_mode_t {
+	char *mode;
+	char *allowed;
+	int skip_m;
+};
+
+static const struct verify_mode_t mode_paras[] = {
```

Comment:
"mode_paras" isn't consistent with the other variables and functions, which call them "mode_params". But this is a nit. You do not have to fix this, but keep it in mind for next time.